bigframes/series.py

```diff
@@ -1216,6 +1222,22 @@ def apply(self, func) -> Series:
         # Reproject as workaround to applying filter too late. This forces the filter
         # to be applied before passing data to remote function, protecting from bad
         # inputs causing errors.
+        if not callable(func):
+            raise ValueError(
+                "Only a ufunc (a NumPy function that applies to the entire Series) or a remote function that only works on single values are supported."
+            )
+
+        if not hasattr(func, "bigframes_remote_function"):
+            try:
+                return func(self)
```

Comment:
Since it may result in incorrect values if this isn't a true vectorized function, let's check for `by_row=False`. If `by_row="compat"` (default) then raise and suggest either remote_function or by_row=False.

Reply:
Done, PTAL.